packages/lit-dev-content/site/docs/libraries/labs.md

```diff
@@ -6,14 +6,26 @@ eleventyNavigation:
+Lit Labs is an umbrella for Lit packages under development that we are actively seeking feedback on. While we encourage real-world use in order to help the feedback process, please note the following important aspects of Labs packages:
 
-Current Lit Labs projects include:
+- Lit Labs projects are published under the `@lit-labs` npm scope.
+- Breaking changes are likely to occur more frequently than in non-labs packages, but they will still respect standard semantic versioning guildelines and all changes will be published to the CHANGELOG files.
+- While bugs in non-lab packages are typically fixed with higher priority, we still treat labs bugs as high priority, and both are triaged as part of the same process.
+- When a Lit Labs project is ready to be graduated, it will begin publishing under the `@lit` scope—for example, the [`@lit/localize`](/docs/localization/overview/) package started out as a Lit Labs project. Once a package graduates, its first version under the `@lit` scope will match that of the latest in `@lit-labs` but only the `@lit` version will receive new updates.
+- We may decide to deprecate a Lit Labs project. In such cases, we wil notify the community and a deprecation warning will be added to the npm package. It will receive bug fix support for at least 6 months. A record of historical labs packages will be kept on this page.
 
-* [Motion](https://github.com/lit/lit/blob/main/packages/labs/motion/README.md#lit-labsmotion). Animation helpers for Lit templates. The `@lit-labs/motion` package includes an `animate` directive for adding tweening animations when an element changes state, as well as an `AnimateController` class for programmatically controlling and coordinating animations. You can find a number of motion examples in the [Playground](https://lit.dev/playground/#sample=examples/motion-simple).
-* [React](https://github.com/lit/lit/tree/main/packages/labs/react#lit-labsreact). React integration helpers for custom elements and reactive controllers.
-* [scoped-registry-mixin](https://github.com/lit/lit/tree/main/packages/labs/scoped-registry-mixin#lit-labsscoped-registry-mixin) Mixin for Lit that integrates with the speculative [Scoped CustomElementRegistry polyfill](https://github.com/webcomponents/polyfills/tree/master/packages/scoped-custom-element-registry).
-* [SSR](https://github.com/lit/lit/tree/main/packages/labs/ssr#lit-labsssr). A package for server-side rendering Lit templates and components.
-* [Task](https://github.com/lit/lit/blob/main/packages/labs/task/README.md#lit-labstask). A reactive controller for handling asynchronous tasks.
+Feedback is currently being solicited on the following Labs packages:
 
-When a Lit Labs project is ready for production it may be migrated to a new location outside of Lit Labs. For example, the [`@lit/localize`](/docs/localization/overview/) package started out as a Lit Labs project.
+| Package | Description | Docs | Feedback |
```

Comment:
Do we want to have the status indicator in this table too? @kevinpschaaf

Reply:
Yeah, like a 🔴 🟠 🟡 🟢 status with a legend maybe?

Reply:
yep

Reply:
sgtm and then maybe sorting them by closest to stable?

Reply:
Status suggestions:

🟢 = Production
🟡 = Evaluation
🟠 = Development
🔴 = At risk (do we need this?)

🟢 react
🟢 ssr
🟢 context
🟢 scoped-registry-mixin
🟡 task
🟡 virtualizer
🟡 motion
🟡 observers
🟡 eleventy-plugin-lit
🟠 router
🟠 testing

Reply:
Is "Production" the right description? Perhaps that is to say it's close to graduation. I don't think ssr belongs in that category though. That feels more like "Development" based on the potential for API changes.

Reply:
Re:
> 🟢 = Production
> 🟡 = Evaluation
> 🟠 = Development
> 🔴 = At risk (do we need this?)

I don't think "production" makes sense in a labs context at all, and this makes me reconsider whether we even want to try to carve out specific words here with meaning, more than just a spectrum of "stability":

🟢 = Most stable, soliciting final feedback
🟡 = ..
🟠 = ..
🔴 = Least stable, soliciting initial feedback

For example, SSR is _not_ green at this point; there are major critical features like async rendering and event support that could plausibly require disruptive changes. However otoh, it's pretty usable, so for those reasons, I would put it in the "yellow" bucket. But does that mean we're done with "development" and in the "evaluation" phase? No, I don't think so. So I'm not sure how helpful it will be to pin ourselves down to a set of concrete gates like that, rather than just a spectrum of stability.

Reply:
I mostly like this, except I would probably say that 🔴 should be something like "stopped or at risk of being stopped". So green is moving towards graduating and red is the opposite. Packages would start in 🟠  usually.

Reply:
Yeah that makes sense, although a bit nuanced for a legend. Maybe @augustjk can take a crack at what the legend would look like.

Reply:
I'll try to come up with something. For the labels though, how about this:

🟢 = Near graduation
🟡 = Somewhat stable (?)
🟠 = Soliciting initial feedback
🔴 = At risk of deprecation / Deprecated (?)

Though I'm wondering if the distinction between yellow and orange are really necessary and perhaps we might just combine them to a single "Under active development" ?

Reply:
I tried splitting up the table into subsections rather than adding an icon to each package and having to label them. WDYT?

<img width="795" alt="image" src="https://user-images.githubusercontent.com/40413829/189777373-fa1b4dd7-f96d-4221-b14a-b4922edfc50e.png">

Open to suggestion on better subheadings.

Reply:
Preview: https://pr930-5f6cea0---lit-dev-5ftespv5na-uc.a.run.app/docs/libraries/labs/
Styling of subheading is the same as the table here: https://lit.dev/docs/templates/directives/

Reply:
I like this format